spectrum-network/src/peer_manager/data.rs

```diff
@@ -37,17 +37,17 @@ impl Serialize for PeerDestination {
 }
 
 impl<'de> Deserialize<'de> for PeerDestination {
-    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    fn deserialize<D>(_deserializer: D) -> Result<Self, D::Error>
     where
         D: Deserializer<'de>,
     {
         todo!()
     }
```

Comment:
Please make sure to sync with the original branch, this Deserialize is implemented there already so no need for `_deserializer`

Reply:
Rebased.